### PR TITLE
Add suggested changelog to make-release.sh

### PR DESCRIPTION
**Pull request by pierremtb**

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -67,3 +67,12 @@ echo "Versions has been bumped in relevant json files, a branch has been created
...
 echo ""
+
+echo "Suggested changelog:"
+echo "\`\`\`"
+echo "## What's Changed"
+git log $(git describe --tags --abbrev=0)..HEAD --oneline --pretty=format:%s | grep -v Bump | grep -v 'Cut release v' | awk '{print "* "toupper(substr($0,0,1))substr($0,2)}'
+echo ""
+echo "**Full Changelog**: https://github.com/KittyCAD/modeling-app/compare/${latest_tag}...${new_version}"
+echo "\`\`\`"
+echo "and would recommend removing ones that aren't related to the product (eg. CI changes)"
```
Comment (Irev-Dev) [inline make-release.sh:78]:
```suggestion
echo "and would recommend removing ones that aren't related to the product (eg. CI or dependabot changes)"
```

Comment (pierremtb) [reply make-release.sh:78]: So with `grep -v Bump` in the command we should be filtering out dependabot changes already, right?

Comment (Irev-Dev) [reply make-release.sh:78]: Oh nice sorry

Comment (pierremtb) [reply make-release.sh:78]: No sweat, appreciate the review

**Pull request merged**